fixtures/blank/issue.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  issue: {
+    body: undefined,
```

Comment:
nitpick: it's set to `null`, not `undefined`
```suggestion
    body: null,
```